### PR TITLE
Enable strict mode

### PR DESCRIPTION
**Pull request by Mogztter**

### CODE REVIEW

File: lib/opal/cli_runners/chrome.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ CDP(options, function(client) {
     Runtime.exceptionThrown(function(exception) {
       var exceptionDetails = exception.exceptionDetails,
           properties = exceptionDetails.exception.preview.properties,
-          stackTrace = exceptionDetails.stackTrace.callFrames,
+          stackTrace = exceptionDetails.stackTrace ? exceptionDetails.stackTrace.callFrames : [],
```
Comment (Mogztter) [inline lib/opal/cli_runners/chrome.js]: `exceptionDetails.stackTrace` can be undefined.
I should probably create a dedicated pull request with just this change, wdyt @elia ?

Comment (Mogztter) [reply lib/opal/cli_runners/chrome.js]: Done :smile:

**Comment by Mogztter** [comment]
I have yet to find the solution but I've opened a few easy to review pull requests to prepare the ground (mostly one line fixes)

**Comment by janbiedermann** [comment]
In PR#1832, Opal.in_strict_mode is available, a boolean, which allows during runtime for checking if running in strict mode, maybe that's useful
https://github.com/opal/opal/pull/1832/commits/ecad156522765627c29cd441ebeef9d00325859a

**Comment by Mogztter** [comment]
I believe that this pull request has been superseded. Should I close?

**Comment by janbiedermann** [comment]
Yes please.

**Comment by janbiedermann** [comment]
Ooops, that wasnt my PR, sorry.

**Pull request closed without merging**